koji-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora
+FROM registry.fedoraproject.org/fedora:33
```

Comment:
I noticed that the builds are trying to call `pip3.7`, which doesn't exist in F33. You can fix that by updating/removing the `PY_VER_*` env variables in the koji containers.

Reply:
OK, we're going ahead with removing Py2 and RHEL < 8